tokens/alias.text.dark.js

```diff
@@ -6,12 +6,24 @@
 module.exports = {
   text: {
     "color"             : { value: "{color.grey.0.value}", attributes: { category: "color" } },
-    "color-weak"        : { value: "{color.grey.40.value}", attributes: { category: "color" } },
-    "color-x-weak"      : { value: "{color.grey.50.value}", attributes: { category: "color" } },
+    "color-hover"             : { value: "{color.grey.0.value}", attributes: { category: "color" } },
+    "color-pressed"             : { value: "{color.grey.0.value}", attributes: { category: "color" } },
```

Comment:
Hover and pressed are the same value?

Reply:
According to design, yes. I added the tokens anyway just in case it may change in the future.